packages/block-editor/src/components/block-canvas/style.scss

```diff
@@ -4,4 +4,7 @@ iframe[name="editor-canvas"] {
 	height: 100%;
 	display: block;
 	background-color: transparent;
+	// Handles transitions between device previews
+	transition: all 400ms cubic-bezier(0.46, 0.03, 0.52, 0.96);
+	@include reduce-motion("transition");
```

Comment:
This should probably be re-added as a mixin or sass variable since it's used in multiple places again.

For reference, this is what we had before.

```scss
@mixin editor-canvas-resize-animation() {
	transition: all 0.4s cubic-bezier(0.46, 0.03, 0.52, 0.96);
	@include reduce-motion("transition");
}
```

Reply:
We overwrite it though, as we need to add more conditions to the end. I could having the mixin for just the timing. But if we add that mixin again, we'd still have to repeat the `all 0.4s cubic-bezier(0.46, 0.03, 0.52, 0.96)` part.

Reply:
For example, if we added the mixin back, the other place that uses that code would look like:

```
	
	@include editor-canvas-resize-animation;
	$zoomOutAnimation: all 400ms cubic-bezier(0.46, 0.03, 0.52, 0.96);
        // overwrite the mixin because we need to use it here.
	transition: $zoomOutAnimation, transform 0s translate 0s scale 0s;
	@include reduce-motion("transition");

	&.zoom-out-animation {
		// we only want to animate the scaling when entering zoom out. When sidebars
		// are toggled, the resizing of the iframe handles scaling the canvas as well,
		// and the doubled animations cause very odd animations.
		transition: $zoomOutAnimation, transform 0s;
	}
```

Reply:
I'll add additional rules to the mixin.